Documentation/helm-operator.md

```diff
@@ -148,6 +154,9 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.volumeReplication.image`       | Volume Replication Controller image.                                                                                        | `quay.io/csiaddons/volumereplication-operator:v0.3.0`     |
 | `csi.csiAddons.enabled`     | Enable CSIAddons                                                                                                  | `false`                                                   |
 | `csi.csiAddons.image`       | CSIAddons Sidecar image.                                                                                        | `quay.io/csiaddons/k8s-sidecar:v0.2.1`     |
+| `csi.nfs.enabled`     | Enable nfs driver.                                                                                                  | `false`                                                   |
+| `csi.nfs.image`       | NFS nodeplugin image. image.                                                                                        | `mcr.microsoft.com/k8s/csi/nfs-csi:v3.1.0`     |
+| `csi.nfs.probe.image`       | NFS nodeplugin liveness probe image. image.                                                                                        | `k8s.gcr.io/sig-storage/livenessprobe:v2.5.0`     |
```

Comment:
do we need livenessproble image from kubernetes-csi?

Reply:
> do we need livenessproble image from kubernetes-csi?

Yes, the official nodeplugin example uses it 
https://github.com/kubernetes-csi/csi-driver-nfs/blob/1177e9be8a0e70648d1bcc9db6b82911317d90a0/deploy/v3.1.0/csi-nfs-node.yaml#L30 

livenessprobe is also under apache 2.0, so we should fine ? 
https://github.com/kubernetes-csi/livenessprobe/blob/master/LICENSE

Reply:
should be fine but this is not useful unless the probe is defined https://github.com/kubernetes-csi/csi-driver-nfs/blob/1177e9be8a0e70648d1bcc9db6b82911317d90a0/deploy/v3.1.0/csi-nfs-node.yaml#L99-L106 and provide an option to enable/disable this one.

Reply:
> should be fine but this is not useful unless the probe is defined https://github.com/kubernetes-csi/csi-driver-nfs/blob/1177e9be8a0e70648d1bcc9db6b82911317d90a0/deploy/v3.1.0/csi-nfs-node.yaml#L99-L106 and provide an option to enable/disable this one.

okay :+1:  default should be enalbled or disabled ?

Reply:
let's disable it by default and if required users can enable it. as its disabled less resource utilization.

Reply:
How necessary is the liveness probe to our initial developmet? It seems easier to do this initial development without the livenessprobe at all, and we can add it later if needed.

Reply:
removing liveness probe for now